kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example

```diff
@@ -47,3 +47,10 @@ instances:
     ## Set a timeout for the prometheus query.
     #
     # prometheus_timeout: 10
+
+    ## @param telemetry_enable - boolean - optional - default: false
+    ## To enable the telemetry check's metrics,  you must set this parameter to true.
+    ## It will generate useful internal check metrics: message payload size, the number
+    ## of metrics received, processed, ignored....
```

Comment:
we could give an indication about how to find them (under <namespace>.telemetry) here